apps/masterbots.ai/components/routes/chat/chat-chatbot-details.tsx

```
@@ -53,18 +56,25 @@ export default function ChatChatbotDetails() {
         flex flex-col gap-[10px] relative font-mono"
       >
         <div className="w-[70%] flex flex-col gap-[10px] px-[24px] pt-[24px]">
-          <div className="text-2xl font-black">
-            {activeChatbot ? activeChatbot?.name : 'Welcome to Masterbots!'}
+         <div className="text-2xl font-black">
+            { page != 'profile' ? (activeChatbot  ? activeChatbot?.name : 'Welcome to Masterbots!') : `Browse ${slug}'s threads`}
```

Biome

[error] Use !== instead of !=. != is only allowed when comparing against `null`

!= is only allowed when comparing against null

Using != may be unsafe if you are relying on type coercion
Unsafe fix: Use !==


(lint/suspicious/noDoubleEquals)